source/beamline/BioXAS/BioXASSSRLMonochromatorRegionControl.cpp

```diff
@@ -116,195 +116,167 @@ void BioXASSSRLMonochromatorRegionControl::setUpperSlitBladeControl(AMControl *n
+                removeChildControl(paddle_);
 
 		paddle_ = paddle;
 
-		if (paddle_)
-			addChildControl(paddle_);
+                addChildControl(paddle_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setPaddleStatusControl(AMControl *paddleStatus)
 {
 	if (paddleStatus_ != paddleStatus) {
 
-		if (paddleStatus_)
-			removeChildControl(paddleStatus_);
+                removeChildControl(paddleStatus_);
 
 		paddleStatus_ = paddleStatus;
 
-		if (paddleStatus_)
-			addChildControl(paddleStatus_);
+                addChildControl(paddleStatus_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setKeyStatusControl(AMControl *keyStatus)
 {
 	if (keyStatus_ != keyStatus) {
 
-		if (keyStatus_)
-			removeChildControl(keyStatus_);
+                removeChildControl(keyStatus_);
 
 		keyStatus_ = keyStatus;
 
-		if (keyStatus_)
-			addChildControl(keyStatus_);
+                addChildControl(keyStatus_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setBrakeStatusControl(AMControl *brakeStatus)
 {
 	if (brakeStatus_ != brakeStatus) {
 
-		if (brakeStatus_)
-			removeChildControl(brakeStatus_);
+                removeChildControl(brakeStatus_);
 
 		brakeStatus_ = brakeStatus;
 
-		if (brakeStatus_)
-			addChildControl(brakeStatus_);
+                addChildControl(brakeStatus_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setBraggControl(AMControl *bragg)
 {
 	if (bragg_ != bragg) {
 
-		if (bragg_)
-			removeChildControl(bragg_);
+                removeChildControl(bragg_);
 
 		bragg_ = bragg;
 
-		if (bragg_)
-			addChildControl(bragg_);
+                addChildControl(bragg_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setBraggAtCrystalChangePositionStatusControl(AMControl *inPosition)
 {
 	if (braggAtCrystalChangePositionStatus_ != inPosition) {
 
-		if (braggAtCrystalChangePositionStatus_)
-			removeChildControl(braggAtCrystalChangePositionStatus_);
+                removeChildControl(braggAtCrystalChangePositionStatus_);
 
 		braggAtCrystalChangePositionStatus_ = inPosition;
 
-		if (braggAtCrystalChangePositionStatus_)
-			addChildControl(braggAtCrystalChangePositionStatus_);
+                addChildControl(braggAtCrystalChangePositionStatus_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setCrystalChangeControl(AMControl *crystalChange)
 {
 	if (crystalChange_ != crystalChange) {
 
-		if (crystalChange_)
-			removeChildControl(crystalChange_);
+                removeChildControl(crystalChange_);
 
 		crystalChange_ = crystalChange;
 
-		if (crystalChange_)
-			addChildControl(crystalChange_);
+                addChildControl(crystalChange_);
 	}
 }
 
 void BioXASSSRLMonochromatorRegionControl::setCrystalChangeCWLimitStatusControl(AMControl *limitStatus)
 {
 	if (crystalChangeCWLimitStatus_ != limitStatus) {
 
-		if (crystalChangeCWLimitStatus_)
-			removeChildControl(crystalChangeCCWLimitStatus_);
+                removeChildControl(crystalChangeCCWLimitStatus_);
```

Comment:
@mButlerDM I believe it is a typo. This method is handling *CW* control and there is another method handles the *CCW* control.